content/en/cosign/signing_with_blobs.md

```diff
@@ -1,71 +1,24 @@
 ---
-title: "Working with Blobs"
+title: "Signing with Blobs"
 category: "Cosign"
 position: 109
 ---
 
-Cosign supports signing and verifying standard files and blobs (or binary large objects), in addition to containers.
+Cosign supports signing and verifying standard files and blobs (or binary large objects), in addition to containers. This topic discusses signing.  For invormation on verifying see [verifying](../verifying.md).
```

Comment:
I would recommend the path link vs. the relative link. The static site converts the `.md` files to html, so we shouldn't link to markdown. 

```suggestion
Cosign supports signing and verifying standard files and blobs (or binary large objects), in addition to containers. This topic discusses signing.  For invormation on verifying see [verifying](/cosign/verify).
```

Reply:
OK, fixed.  Also fixed a typo.